crashers_test.go

```diff
@@ -33,7 +33,7 @@ func TestCrash1(t *testing.T) {
 	var s t2
 	err = Decode(in, &s)
 	assert.NotNil(t, err)
-	assert.Equal(t, "Error while decoding field protobuf.t2.T3s: append to non-slice", err.Error())
+	assert.Equal(t, "Error while decoding field {Name:T3s PkgPath: Type:[3]protobuf.t3 Tag: Offset:112 Index:[4] Anonymous:false}: append to non-slice", err.Error())
```

Comment:
Maybe I'm missing something, isn't the type `t2` here?

Reply:
Inside type t2 is a field named T3s, which is the one that we are failing to decode.

Reply:
I was referring to the `t3` of this part `Type:[3]protobuf.t3`, but now I understand because it's referring to the type `t3`, which is the type of the field`T3`.